frontstage_api/resources/passwords/change_password.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from flask import request
+from flask_restplus import Resource
+from structlog import wrap_logger
+
+from frontstage_api import api, auth
+from frontstage_api.controllers import party_controller
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@api.route('/change_password')
+class ChangePassword(Resource):
+
+    @staticmethod
+    @auth.login_required
+    def put():
+        logger.info('Attempting to change user password')
+        message_json = request.get_json(force=True)
```

Comment:
We shouldn't need to use the force parameter. We should be setting the content type correctly in the client calls